src/components/Icon.tsx

```diff
@@ -15,17 +15,27 @@ export const iconNames = [
 	"stacked-column-down",
 	"coins-alt",
 	"calendar",
+	"trend-up-circle",
+	"trend-down-circle",
 ] as const;
 export type IconName = typeof iconNames[number];
 
 interface IconProps {
 	name: IconName;
+	size?: "small" | "middle" | "large";
```

Comment:
maybe it's nicer to just have an enum with the string value being the dimensions or something. there is an requirement for this union to match the property names of the iconDimensions object that is not enforced anywhere